README.md

```diff
@@ -154,32 +154,68 @@ To **save you from coding boilerplate classes** we created **Annotation Processo
 
 *Notice that annotation processors are not part of the library core, you can work with StorIO without them, we just made them to save you from boilerplate*.
 
+`StorIOSQLite`:
 ```groovy
 dependencies {
-	// At the moment there is annotation processor only for StorIOSQLite 
 	compile 'com.pushtorefresh.storio:sqlite-annotations:insert-latest-version-here'
 
 	// We recommend to use Android Gradle Apt plugin: https://bitbucket.org/hvisser/android-apt
 	apt 'com.pushtorefresh.storio:sqlite-annotations-processor:insert-latest-version-here'
 }
 ```
 
+`StorIOContentResolver`:
+```groovy
+dependencies {
+	compile 'com.pushtorefresh.storio:sqlite-annotations:insert-latest-version-here'
```

Comment:
`storio-content-resolver-annotations` and `storio-content-resolver-annotations-processor`